### PR TITLE
Disable CSRF for add-in

### PR DESCRIPTION
**Pull request by oliverguenther**
The TaskConnector add-in doesn't handle CSRF tokens.

https://community.openproject.org/work_packages/22390/activity

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://oliverguenther-fix-v2-addin-csrf.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://oliverguenther-fix-v2-addin-csrf.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

File: app/controllers/api/v2/planning_elements_controller.rb [hunk 1]
```diff
@@ -32,6 +32,9 @@ module V2
     class PlanningElementsController < ApplicationController
       helper :timelines, :planning_elements
 
+      # disable CSRF protection since the add-in doesn't support it
+      skip_before_filter :verify_authenticity_token
```
Comment (ulferts) [inline app/controllers/api/v2/planning_elements_controller.rb]: Why is that statement only added here and not on the other APIv2 controllers as well?

**Comment by oliverguenther** [comment]
We'll likely need more but I don't know the exact accessors of the APIv2 in TaskConnector and don't want to whitelist all v2 controllers. I have asked for support by an active user of the add-in to gather the required routes.

**Comment by ulferts** [comment]
LGTM. Free to merge once user reports it to be working with the add in.

**Pull request merged**